services/brig/src/Brig/App.hs

```diff
@@ -231,8 +231,12 @@ newEnv o = do
     SqsQueue q -> SqsQueue <$> AWS.getQueueUrl (aws ^. AWS.amazonkaEnv) q
   mSFTEnv <- mapM Calling.mkSFTEnv $ Opt.sft o
   prekeyLocalLock <- case Opt.randomPrekeys o of
-    Just True -> Just <$> newMVar ()
-    _ -> pure Nothing
+    Just True -> do
+      Log.warn lgr $ Log.msg (Log.val "randomPrekeys: active")
+      Just <$> newMVar ()
+    _ -> do
+      Log.warn lgr $ Log.msg (Log.val "randomPrekeys: not active; using dynamoDB instead.")
```

Comment:
why warn, and not info?  (i trust you, just wondering.)

Reply:
no good reason, I changed this to Info, that's better, thanks!